SNOW-532795: Bumped up PythonConnector PATCH version from 2.7.2 to 2.… (#1033)
snowflakedb/snowflake-connector-python: DESCRIPTION.md

# Release Notes

- v2.7.3(January 22,2022)

   - shifted package metadata from setup.py to setup.cfg
   - Add Timezone to Timestamp_TZ 
   - Fix Storage Credential Error
   - fix py.typed not being included in wheels
   - Negative Numbers Mangled for arrow_number_to_decimal
   - Add GET Empty File Handling
   - fix long description rendering for PyPi
   - handling no sms case for DUO

>    - Update requirements files Update numpy requirement from <1.22.0 to <1.23.0

Reviewer: Can we add this line:

"Add the ability to auto-create a table when writing a pandas DataFrame to a Snowflake table"

# Release Notes

- v2.7.3(January 22,2022)

>    - shifted package metadata from setup.py to setup.cfg

Reviewer 2: ```suggestion
```

>    - Add Timezone to Timestamp_TZ 

Reviewer 2: ```suggestion
   - Fixed a bug where timezone was missing from retrieved Timestamp_TZ columns
```

>    - Fix Storage Credential Error

Reviewer 2: ```suggestion
   - Fixed a bug where a long running PUT/GET command could hit a Storage Credential Error while renewing credentials
```

>    - fix py.typed not being included in wheels

Reviewer 2: ```suggestion
   - Fixed a bug where py.typed was not being included in our release wheels
```

>    - Negative Numbers Mangled for arrow_number_to_decimal

Reviewer 2: ```suggestion
   - Fixed a bug where negative numbers were mangled when fetched with the connection parameter arrow_number_to_decimal
```

>    - Add GET Empty File Handling

Reviewer 2: ```suggestion
   - Improved the error message that is encountered when running GET for a non-existing file
```

>    - fix long description rendering for PyPi

Reviewer 2: ```suggestion
   - Fixed rendering of our long description for PyPi
```

>    - handling no sms case for DUO

Reviewer 2: ```suggestion
   - Fixed a bug where DUO authentication ran into errors if sms authentication was disabled for the user
```

   - Add the ability to auto-create a table when writing a pandas DataFrame to a Snowflake table

>    - Update requirements files Update numpy requirement from <1.22.0 to <1.23.0

Reviewer 2: ```suggestion
   - Bumped the maximum dependency version of numpy from <1.22.0 to <1.23.0
```

>    - Update CODEOWNERS

Reviewer 2: ```suggestion
```
